api-client/package.json

```diff
@@ -17,13 +17,13 @@
   },
   "dependencies": {
     "entities": "^2.0.0",
+    "isomorphic-dompurify": "^0.14.0",
```

Comment:
Why aren't we using dompurify directly?

Reply:
So I understood @Taggotty wrong, she just said that maybe `dompurify` does not work on native but I thought she checked and it does not work. However, it does work, so we'll use `dompurify` directly.